LocationServices/LocationServices/Scenes/Tracking/ViewModel/TrackingViewModel.swift

```diff
@@ -104,9 +114,14 @@ final class TrackingViewModel: TrackingViewModelProtocol {
                 NotificationCenter.default.post(name: Notification.updateTrackingHistory, object: self, userInfo: ["history": history])
                 self?.delegate?.drawTrack(history: history)
             case .failure(let error):
-                let model = AlertModel(title: StringConstant.error, message: error.localizedDescription, cancelButton: nil)
-                DispatchQueue.main.async {
-                    self?.delegate?.showAlert(model)
+                if(ErrorHandler.isAWSStackDeletedError(error: error)) {
```

Comment:
This is duplicated all over the code. Can we move it to the ErrorHandler?
we can have function showError() and we will be able to move validation there